Transformations: Add unary operations to Add field from calculation (#75946)
grafana/grafana: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

- **Mode -** Select a mode:
  - **Reduce row -** Apply selected calculation on each row of selected fields independently.

>   - **Binary operation -** Apply basic binary operations(sum, multiply, etc) on values in a single row from two selected fields.

Reviewer: This is maybe a bit beyond the context of this PR but should it seems like we should include language about what these operators are. I'm assuming plenty of folks will know, but I'm guessing there are those that may be confused. What do you think?

Author: I agree! I've added explanations to each of the new functions added. I think binary operations should be fine as they are since they are pretty self explanatory. @imatwawana can I please ask for another review of the docs with these new modifications? Thank you!

>   - **Binary operation -** Apply basic binary operations(sum, multiply, etc) on values in a single row from two selected fields.
>   - **Unary operation -** Apply basic unary operations(abs, exp, floor, etc) on values in a single row from a selected field.

Reviewer 2: ```suggestion
  - **Binary operation -** Apply basic binary operations (for example, sum or multiply) on values in a single row from two selected fields.
  - **Unary operation -** Apply basic unary operations (for example, abs, exp, or floor) on values in a single row from a selected field.
```

  - **Row index -** Insert a field with the row index.
- **Field name -** Select the names of fields you want to use in the calculation for the new field.
- **Calculation -** If you select **Reduce row** mode, then the **Calculation** field appears. Click in the field to see a list of calculation choices you can use to create the new field. For information about available calculations, refer to [Calculation types][].

> - **Operation -** If you select **Binary operation** or **Unary operation** mode, then the **Operation** fields appear. These fields allow you to do basic math operations on values in a single row from selected fields. You can also use numerical values for binary operations.

Reviewer 2: ```suggestion
- **Operation -** If you select **Binary operation** or **Unary operation** mode, then the **Operation** fields appear. These fields allow you to apply basic math operations on values in a single row from selected fields. You can also use numerical values for binary operations.
```

- **Mode -** Select a mode:
  - **Reduce row -** Apply selected calculation on each row of selected fields independently.
  - **Binary operation -** Apply basic binary operations (for example, sum or multiply) on values in a single row from two selected fields.
  - **Unary operation -** Apply basic unary operations on values in a single row from a selected field. The available operations are:

>     - **Absolute value (abs)** - Returns the absolute value of the given expression. It represents it's distance from zero as a positive number.
>     - **Natural exponential (exp)** - Returns _e_ raised to the power of the given expression.
>     - **Natural logarithm (ln)** - Returns the natural logarithm of a given expression.
>     - **Floor (floor)** - Returns the largest integer less than or equal to the given expression.
>     - **Ceiling (ceil)** - Returns the smallest integer greater than or equal to the given expression.

Reviewer 2: ```suggestion
    - **Absolute value (abs)** - Returns the absolute value of a given expression. It represents its distance from zero as a positive number.
    - **Natural exponential (exp)** - Returns _e_ raised to the power of a given expression.
    - **Natural logarithm (ln)** - Returns the natural logarithm of a given expression.
    - **Floor (floor)** - Returns the largest integer less than or equal to a given expression.
    - **Ceiling (ceil)** - Returns the smallest integer greater than or equal to a given expression.
```
